.github/workflows/ci-build-push.yml

```diff
@@ -0,0 +1,34 @@
+name: "build-push"
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Looks like pushing images of master to dockerhub, we may need to do it in gitlab. 

And it is totally ok to build everything in gitlab and publish to it's container registry. Only GITHUB_TOKEN will be needed and it will be available for every branch and PR. And developers may get it from github container registry. https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

But for master builds i am in favor of using gitlab due to secret exposure.

Reply:
Hey @pfltdv. I totally agree. 

This aims to build and publish temporary artifacts to speedup/unblock development for the project. 
To be removed after we conclude with templating & finalizing our build and release process

Reply:
Thanks @phoinixgrr , they resolving this comment.